modBoot/src/org/aion/Aion.java

```diff
@@ -92,16 +93,27 @@ public static void main(String args[]) {
             throw e;
         }
 
-        /* Outputs relevant logger configuration */
+        // UUID check
+        String UUID = cfg.getId();
+        if (!UUID.matches("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")) {
+            System.out.println("Invalid UUID; please check <id> setting in config.xml");
+            exit(-1);
+        }
+
+        // Log/Database path
         if (!cfg.getLog().getLogFile()) {
             System.out
                 .println("Logger disabled; to enable please check log settings in config.xml\n");
         } else if (!cfg.getLog().isValidPath() && cfg.getLog().getLogFile()) {
             System.out.println("File path is invalid; please check log setting in config.xml\n");
             return;
         } else if (cfg.getLog().isValidPath() && cfg.getLog().getLogFile()) {
-            System.out.println("Logger file path: '" + cfg.getLog().getLogPath() + "'\n");
+            System.out.println("Logger path: '" + cfg.getLog().getLogPath() + "'");
         }
+        System.out.println("Database path: '" + cfg.getDb().getPath() + "'");
```

Comment:
Please add the full path for the logger and the db. To get it, append the value you have now to the base path.

Reply:
sure